samples/cpp/stitching_detailed.cpp

```diff
@@ -813,7 +817,7 @@ int main(int argc, char* argv[])
             }
             blender->prepare(corners, sizes);
         }
-        else if (!timelapser)
+        else if (!timelapser && timelapse)
```

Comment:
Right. Agreed with fix.
Lines 795+816 works as expected, but it is for first iteration only (there is a for loop). Application with disabled timelapse mode was assert message.